sql-plugin/src/main/311until340-all/scala/org/apache/spark/sql/rapids/RapidsSchemaUtils.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids
+
+import org.apache.spark.sql.catalyst.analysis.Resolver
+import org.apache.spark.sql.types.StructType
+import org.apache.spark.sql.util.SchemaUtils
+
+object RapidsSchemaUtils {
```

Comment:
If you are fixing a compile error like this, please pull it out into a separate PR.  This is something that will benefit everyone by unblocking the build and should not be combined with and blocked by an unrelated change going in.

Reply:
Is this still needed now that https://github.com/NVIDIA/spark-rapids/pull/7153 is merged in?